client/main.lua

```diff
@@ -280,7 +280,9 @@ engineBind = lib.addKeybind({
     name = 'toggleengine',
     description = locale('info.engine'),
     defaultKey = 'G',
-    disabled = not cache.vehicle,
+    disable = function ()
+        return not cache.vehicle
+    end,
```

Comment:
disable is not a param of addKeybind. https://overextended.dev/ox_lib/Modules/AddKeybind/Client#libaddkeybind

Reply:
```suggestion
    disabled = function ()
        return not cache.vehicle
    end,
```